### DISCUSSION TITLE
[Feature] File location data

### DISCUSSION THREAD

**Discussion by DMcWetty** [immich-app/immich#11248] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Sorry if I overlooked this feature request in the search (there are a lot of requests, haha). 

I'd like to see the file location (where the file physically resides on disk) within the information tab. I have most of my photos uploaded from my iPhone/iCloud. But I have an external library with duplicates that I've accumulated over the years. I'd love to be able to track down where these dupes live on disk and remove the offending copies.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Click the (i) button next to the filename, it'll show the path.

**Reply by floydheld** [reply]
it shows filename only

**Comment by moinmoin** [comment]
@bo0tzz the (i) button does not appear, if the file is shared. Especially in case of external libraries, which are more likely shared, the location of the file would still be interesting.

**Reply by JeffPeters** [reply]
I found you can get the data if re-write the url:

For example if the url is: 
```
http://<host>:<port>/partners/<partner_id>/photos/<photo_id>
http://127.0.0..1:2283/partners/c500f1f9-dc15-40a7-8b9b-b4ff3f33c93d/photos/213ea8c6-da1f-4320-9d90-e82967e93fa6
```
you can make a request for:
```
http://<host>:<port>/api/assets/<photo_id>
http://127.0.0.1:2283/api/assets/685478df-9ab2-448a-b515-afc87d79f366
```
and it will return json data which includes originalPath.
```
...,"originalPath":"/mnt/...
```

So the data is accessable by the other user. Just need to update some javascript to add the data after making the extra request.

Sometimse the main page this data is also given from the websocket.
